utils/daily.go

```diff
@@ -0,0 +1,89 @@
+package utils
+
+import (
+	"cronicle/ui/constants"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"time"
+)
+
+type WriteDailyParams struct {
+	Message, Tags, Date string
+}
+
+func GetDate() string {
+	time := time.Now()
+	return time.Format("2006-01-02")
+}
+
+func AddToCurrentDaily(w WriteDailyParams) {
+	path := GetPath([]string{"daily", GetDate() + ".md"})
+	if _, err := os.Stat(path); err == nil {
+		// current daily file exists
+		AddEntry(w)
+	} else if errors.Is(err, os.ErrNotExist) {
+		// no daily file created yet
+		createdDateTime := time.Now().Format("2006-01-02 15:04")
+		m := fmt.Sprintf("- %s\n", w.Message)
+		newDaily := ComposeDaily(WriteDailyParams{Message: m, Tags: w.Tags, Date: createdDateTime})
+		WriteToFile(newDaily, GetPath([]string{"daily", GetDate() + ".md"}))
+	} else {
+		log.Fatal(constants.ERROR_WRITE_FILE, err)
+	}
+
+}
+
+func AddEntry(w WriteDailyParams) {
```

Comment:
I believe this can be a private function?

```suggestion
func AppendEntry(w WriteDailyParams) {
```